ietf/templates/submit/api_submit_info.html

```diff
@@ -27,7 +27,6 @@ <h1 class="mb-3">Internet-Draft submission API instructions</h1>
     </p>
     <ul>
         <li>Only XML-only uploads are supported, not text or combined.</li>
-        <li>Document replacement information cannot be supplied.</li>
```

Comment:
This one needs to stay - the replaces is only supported through the asynchronous endpoint.

Reply:
Ah good catch, thanks! Fixed.